src/components/AllCameraimg.jsx

```diff
@@ -0,0 +1,30 @@
+import Webcam from "react-webcam";
```

Comment:
I can't find where this component is being used.

Reply:
it is being used on line 21 in the same file.

Reply:
I meant allcameraimg component

Reply:
Thankyou for your review @narayan954 , removed allcameraimg component. Pls do check.

Reply:
> Thankyou for your review @narayan954 , removed allcameraimg component. Pls do check.

Yeah sure, allow me some time, I'll do by tonight